### PR TITLE
core-svc-feat(document-archive): add endpoint tab status

### PR DESCRIPTION
**Pull request by ayocodingit**
## Overview
- core-svc-feat(document-archive): add endpoint tab status

## Evidence:
project: Portal Jabar
title: core-svc-feat(document-archive): add endpoint tab status
participants: @ayocodingit @rachadiannovansyah

### CODE REVIEW

File: core-service/src/domain/document_archive.go [hunk 1]
```diff
@@ -43,12 +43,18 @@ type DocumentArchiveRequest struct {
 	Category    string `json:"category" validate:"required"`
 }
 
+type DocumentArchiveTabStatus struct {
+	Status string `json:"status"`
+	Count  int    `json:"count"`
+}
+
```
Comment (rachadiannovansyah) [inline core-service/src/domain/document_archive.go:50]: the `tabs status response` struct is already available [here](https://github.com/jabardigitalservice/portal-jabar-services/blob/fe68a21ef2b280ab3f700b1075bffdeb4dcc2c6b/core-service/src/domain/news.go#L150) 

so you can re-use the existing struct for the same requirements 👍

**Review by rachadiannovansyah — CHANGES_REQUESTED** [review]

File: core-service/src/domain/document_archive.go (hunk 1 above)
Comment (ayocodingit) [reply core-service/src/domain/document_archive.go:50]: Yes, but how if we on principle Clean Architecture ? so i create new variable on domain document-archive, wdyt? 😃

Comment (rachadiannovansyah) [reply core-service/src/domain/document_archive.go:50]: i see, this principle is separated domain by domain requirement but in this case, we have the same struct attribute on each domain and I just wanna try to make it not redundant. 
Hmm interesting, how about we make the global domain to put the re-useable structs here 🤔

Comment (rachadiannovansyah) [reply core-service/src/domain/document_archive.go:50]: to be discussed later.. by the way, it is safe to merge. letsgo

Comment (ayocodingit) [reply core-service/src/domain/document_archive.go]: Okay, for now, I am done to change to use `TabStatusResponse` thanks for insight :smile:

**Review by rachadiannovansyah — APPROVED** [review]

**Pull request merged**